pkg/domain/entities/images.go

```diff
@@ -318,3 +318,32 @@ type SignOptions struct {
 
 // SignReport describes the result of signing
 type SignReport struct{}
+
+// ImageMountOptions describes the input values for mounting containers
+// in the CLI
+type ImageMountOptions struct {
+	All        bool
+	Format     string
+	NoTruncate bool
+}
+
+// ImageUnmountOptions are the options from the cli for unmounting
+type ImageUnmountOptions struct {
+	All   bool
+	Force bool
+}
+
+// ImageMountReport describes the response from container mount
+type ImageMountReport struct {
+	Err          error
+	Id           string //nolint
+	Name         string
+	Repositories []string
+	Path         string
+}
+
+// ImageUnmountReport describes the response from umounting a container
+type ImageUnmountReport struct {
+	Err error
+	Id  string //nolint
```

Comment:
Why not ID?

Reply:
Cut and paste from container code. but I will change to ID.